lib/PlaidClient.js

```diff
@@ -479,31 +479,21 @@ Client.prototype.listPaymentRecipients =
 
 // createPayment(String, String, Object, Function)
```

Comment:
This should now be updated to be `createPayment(String, String, Object, Object?, Function)` right?